Plugins/IngameDebugConsole/Scripts/DebugLogConsole.cs

```diff
@@ -439,7 +455,7 @@ private static void AddCommand( string command, string description, string metho
 			AddCommand( command, description, method, instance, parameterNames );
 		}
 
-		private static void AddCommand( string command, string description, MethodInfo method, object instance, string[] parameterNames )
+		public static void AddCommand( string command, string description, MethodInfo method, object instance, string[] parameterNames )
```

Comment:
How about `internal`?

Reply:
For this method and the other mentioned before, making them internal so they can't be called from user code means they won't be able to use these methods if the users make some additional custom Console Attributes.

Reply:
That shouldn't be a problem since there are already Console Attributes for these functions. If a user creates an alternative to ConsoleMethodAttribute, then I'd very much like to hear their reasoning.

When a user types DebugLogConsole.AddCommand to add a command via Scripting API, I want them to use the documented public functions and not these particular functions.